### DISCUSSION TITLE
Vertica database connection not populating under database navigator after successful setup

### DISCUSSION THREAD

**Discussion by qnguyen793** [dbeaver/dbeaver#8947] [Q&A] [answered]
#### System information: 
- Dell Latitude 5480 i5
- DBeaver version 7.1.0

#### Connection specification:
- Vertica 8.1.1

#### Describe your question:
When creating a new database connection after populating fields and clicking test connection it seems the connection is successful. When clicking finish the database is not populating under database navigator. What are steps to take to get the database to populate under database navigator? This method for database setup has worked in the past

**Comment by uslss** [comment]
Hi @qnguyen793 ,
could you check if you create connection in active project?
probably some filter is enabled?

**Comment by qnguyen793** [comment]
Hi @uslss ! Appreciate the reply, I was looking into that. So now I have created a project and was able to make the connection under that project (and queries seem to work)! I was able to make the connection without having to do that before though. Is there a way to have the database under database navigator without having to create a project?

**Comment by uslss** [comment, marked as answer]
@qnguyen793 there is no possibility to create connections without a project. When dbeaver is started at the first time, default project General is automatically created. Connections from active project are shown in DB Navigator.
Active projects could be changed in Projects View (https://github.com/dbeaver/dbeaver/wiki/Projects-View)
